ui/src/app/cron-workflows/components/cron-workflow-details/cron-workflow-details.tsx

```diff
@@ -87,6 +87,99 @@ export const CronWorkflowDetails = ({match, location, history}: RouteComponentPr
+                        .then(() => setError(null))
+                        .catch(setError)
+            },
+            {
+                title: 'Update',
+                iconClassName: 'fa fa-save',
+                disabled: !edited,
+                action: () => {
+                    // magic - we get the latest from the server and then apply the changes from the rendered version to this
+                    return services.cronWorkflows
+                        .get(name, namespace)
+                        .then(latest =>
+                            services.cronWorkflows.update(
+                                {
+                                    ...latest,
+                                    spec: cronWorkflow.spec,
+                                    metadata: {...cronWorkflow.metadata, resourceVersion: latest.metadata.resourceVersion}
+                                },
+                                cronWorkflow.metadata.name,
+                                cronWorkflow.metadata.namespace
+                            )
+                        )
+                        .then(setCronWorkflow)
+                        .then(() => notifications.show({content: 'Updated', type: NotificationType.Success}))
+                        .then(() => setError(null))
+                        .then(() => setEdited(false))
+                        .catch(setError);
+                }
+            },
+            suspendButton,
+            {
+                title: 'Delete',
+                iconClassName: 'fa fa-trash',
+                disabled: edited,
+                action: () => {
+                    popup.confirm('confirm', 'Are you sure you want to delete this cron workflow?').then(yes => {
+                        if (yes) {
+                            services.cronWorkflows
+                                .delete(name, namespace)
+                                .then(() => navigation.goto(uiUrl('cron-workflows/' + namespace)))
+                                .then(() => setError(null))
+                                .catch(setError);
+                        }
+                    });
+                }
+            },
+            {
+                title: 'Share',
+                iconClassName: 'fa fa-share-alt',
+                action: () => setSidePanel('share')
+            }
+        ];
+
+        if (cronWorkflow?.spec?.workflowSpec?.workflowTemplateRef) {
+            const templateName = cronWorkflow.spec.workflowSpec.workflowTemplateRef.name;
+            const clusterScope = cronWorkflow.spec.workflowSpec.workflowTemplateRef.clusterScope;
+            const url: string = clusterScope ? `/cluster-workflow-templates/${templateName}` : `/workflow-templates/${cronWorkflow.metadata.namespace}/${templateName}`;
+
+            const templateLink: Link = {
+                name: 'Workflow Template Link',
```

Comment:
Please change to “Open Workflow Template” and use the same icon as left-hand-nav does.